src/main/java/utils/Parser.java

```diff
@@ -0,0 +1,26 @@
+package utils;
+
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.List;
+import validator.InputValidator;
+
+public class Parser {
+
+    public static List<String> parseStringArray(String names) {
+        List<String> parsedNames = parseStringArrayWithComma(names);
+        InputValidator.validateDuplicate(parsedNames);
+        return parsedNames;
+    }
+
+    private static List<String> parseStringArrayWithComma(String names) {
+        if (names == null || names.isEmpty()) {
+            return new ArrayList<>(); // 빈 배열 반환
```

Comment:
```suggestion
            return Collections.emptyList(); // 빈 배열 반환
```
컬렉션의 정적메서드로도 사용할 수 있어요.
`new ArrayList<>();` 와는 어떤 차이가 있을까요?

Reply:
`Collections.emptyList()`는 `ArrayList`가 아닌 `EmptyList`라는 불변(immutable) 객체를 반환하고 있네요! 또한 메서드 이름만보더라도 빈 리스트를 반환한다는 의도가 잘 전달되는 것 같아요